internal/updater/html/parsing.go

```diff
@@ -0,0 +1,98 @@
+package htmlutils
```

Comment:
Rename this to `html` so it matches the directory name.
In `internal/provider/slickvpn/updater/*.go` files, you can however use a package alias like it is now with

```go
htmlutils "github.com/qdm12/gluetun/internal/updater/html"
```